index.js

```diff
@@ -72,9 +72,34 @@ var browserslist = function (selections, opts) {
         selections = selections.split(/,\s*/);
     }
 
+    var statsFile, statsExist;
+
     if ( opts.stats || process.env.BROWSERSLIST_STATS ) {
+        statsExist = true;
+    } else {
+        (function searchStatsFile(startPath) {
```

Comment:
@AleshaOleg if we process `/a/b/c/file.css` we should check stat file in `/a/b/c`, `/a/b`, `/a` and `/`.

And we already do this “recursion” search in config loading (but without recursion in code).